controllers/user.controller.js

```diff
@@ -34,8 +38,24 @@ const updateUser = async (req, res, next) => {
     }
 };
 
+const uploadAvatar = async (req, res, next) => {
```

Comment:
@ikadushkina Do you handle the case when a user wants to update another user avatar?

Reply:
No